cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -301,18 +289,22 @@ function updateView() {
 
 /**
  * Updates the sentence above the chart
- * @param {string} data - TODO: Add description.
+ * @param {Array} totalVals - List of interest rates.
  */
-function updateLanguage( data ) {
-
+function updateLanguage( totalVals ) {
   function renderLocation() {
-    const state = $( '#location option:selected' ).text();
-    $( '.location' ).text( state );
+    const stateDropDown = document.querySelector( '#location' );
+    const state = stateDropDown.options[stateDropDown.selectedIndex].text;
```

Comment:
Should this be `textContext`?

Reply:
Sure, I think they are equivalent but consistency is good.